spec/factories/organisations.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :organisation do
+    name { "MyString" }
```

Comment:
Fair point, but unless I'm testing the thing specifically then I'm not concerned with what FactoryBot populates properties with. This can be seen in tests where we specifically set model properties under test.

Reply:
Updated